br/pkg/lightning/mydump/csv_parser_test.go

```diff
@@ -680,6 +682,28 @@ func TestConsecutiveFields(t *testing.T) {
 	})
 }
 
+func TestTooLargeRow(t *testing.T) {
+	cfg := config.MydumperRuntime{
+		CSV: config.CSVConfig{
+			Separator: ",",
+			Delimiter: `"`,
+		},
+	}
+	var testCase bytes.Buffer
+	testCase.WriteString("a,b,c,d")
+	// WARN: will take up 120MB memory here.
+	for i := 0; i < tidbconfig.MaxTxnEntrySizeLimit; i++ {
```

Comment:
maybe create a variable in `init` so we can easily test it. Not sure if CI can tolerate this.

Reply:
cd6e96e

Reply:
dc59781